js/components/harvest/job.vue

```diff
@@ -84,6 +84,9 @@ export default {
                 key: 'remote_id',
                 sort: 'remote_id',
                 ellipsis: true
+            }, {
```

Comment:
Maybe we could move `Logs` after Status and add `width` : 100 as is done for status?

Reply:
I'm not sure about that… It'll move status for every line (even if the page doesn't have any logs), with this order, the page without logs will look exactly the same as now. And only item with logs will display the info. But as you wish :-)

Reply:
It was a suggestion because I found the absence of border between fields made it less readable. Could you try with a small width?

Reply:
Done in ec1800133